src/index.js

```diff
@@ -131,7 +131,7 @@ export default async ({ markdownAST }, options = {}) => {
       const $ = cheerio.load(content.div);
       const file = query.file.replace(/[^a-zA-Z0-9_]+/, "-");
       query.highlights.forEach(line => {
-        $(`#file-${file}-LC${line}`).addClass("highlighted");
+        $(`#file-${file.toLowerCase()}-LC${line}`).addClass("highlighted");
```

Comment:
Let's optimize this a bit, shall we?
Move the `.toLowerCase()` call outside the loop.

Line 132 seems like a good place to do this...
```js
const file = query.file.replace(/[^a-zA-Z0-9_]+/g, "-").toLowerCase();
```

Reply:
Looks good, you're right that is a better place for it 🙂